docs/Using A2C.md

```diff
@@ -0,0 +1,45 @@
+Example code to run the algo
+
+```python
+import gym
+
+from genrl import A2C
+from genrl.classical.common import Trainer
+from genrl.deep.common import OnPolicyTrainer
+from genrl.environments import VectorEnv
+
+env = VectorEnv("CartPole-v0")
+agent = A2C(network_type='mlp', env=env, rollout_size=2048)
+trainer = OnPolicyTrainer(agent, env, log_mode=['stdout', 'tensorboard'], log_key="Episode")
+trainer.train()
```

Comment:
Just `A2C("mlp", env, rollout_size=2048)` is enough. You don't normally name the *args.

Over here, there are a few important hyperparameters which you need to list down, not all of them. Mention some that are relevant for tuning the performance. For example: `lr_policy`, `lr_value` and `rollout_size`. Additionally, you can also include `gamma` and `layers`.